src/middlewares.js

```diff
@@ -110,6 +110,11 @@ export function handleParseHeaders(req, res, next) {
   req.config = new Config(info.appId, mount);
   req.info = info;
 
+  const ip = req.ip;
```

Comment:
Actually, express has a [guide](https://expressjs.com/en/guide/behind-proxies.html) for running behind proxies. Allowing it to handle itself internally, `req.ip` would just return what it can deduce as the original client ip. All that would have to be done is to add the following.
```js
app.enable('trust proxy');
```